app/build.gradle

```diff
@@ -47,8 +47,10 @@ dependencies {
     compile 'com.google.firebase:firebase-config:9.8.0'
     compile 'com.android.support:design:23.4.0'
     compile 'org.greenrobot:eventbus:3.0.0'
-    compile 'com.afollestad.material-dialogs:core:0.8.5.9'
+    compile 'com.android.support:design:23.4.0'
+    compile 'com.android.support:cardview-v7:23.4.0'
 
+    compile 'com.afollestad.material-dialogs:core:0.9.1.0'
```

Comment:
Isso força a baixar a SDK 24 do android, já disse! Por quê atualizaram de novo? Tem alguma coisa de muito importante nessa versão?